README.md

```diff
@@ -56,7 +56,7 @@ Once you setup the plugin correctly, you can call the `:generateSwagger` gradle
 
 In order to run this gradle plugin you need to fulfill the following requirements:
 
-* Gradle 5.x - This plugin uses Gradle 5 features, and you will need to setup your Gradle wrapper to use 5.0 or more.
+* Gradle 5.x - This plugin uses Gradle 5 features, and you will need to setup your Gradle wrapper to use 5.4.1 or more.
```

Comment:
I bumped from `5.0` to `5.4.1` because the testProject fails on `5.0` due to:

```
Build file '/home/martin/git/swagger-gradle-codegen/gradle-plugin/plugin/junit7355662627637669365/project/build.gradle.kts' line: 9

* What went wrong:
Script compilation errors:

  Line 09:       content {
                 ^ Unresolved reference: content

  Line 10:         excludeGroup("com.yelp.codegen")
                   ^ Unresolved reference: excludeGroup
```

I think it should be safe to assume most people are using gradle >= 5.4.1 but we can certainly remove that limitation if really needed.